runtime/src/builtins.rs

```diff
@@ -30,15 +30,22 @@ fn genesis_builtins() -> Vec<Builtin> {
     ]
 }
 
+#[derive(Debug, Clone)]
+pub enum ActivationType {
```

Comment:
oh, https://buildkite.com/solana-labs/solana/builds/34410#3d37c5dd-fddb-417c-8cb1-b6a514b0384b/139-1725

Sorry another my picky work is complaining to you again. ;)
Please add AbiExample/AbiEnumVisitor, this is needed for all types accessible from Bank.
These derives are like `Default` or `Debug` for the ABI check magic.